tests/test_features/test_jointplot.py

```diff
@@ -0,0 +1,65 @@
+# tests.test_features.test_jointplot
+# Test the JointPlotVisualizer
+#
+# Author:   Prema Damodaran Roman
+# Created:  
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_jointplot.py
```

Comment:
The ID line is also special and is automatically filled in with the revision when we merge to master. It looks something like this:

```
# ID: setup.py [] benjamin@bengfort.com $
```

You can just put your GitHub username instead of your email if you'd prefer; but technically that's the email associated with the commit that created the document.